### PR TITLE
Added release instructions

### PR DESCRIPTION
**Pull request by sverhoeven**

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -38,7 +39,24 @@ You want to make some kind of change to the code base
 #. make sure the existing tests still work by running ``python setup.py test``;
 #. add your own tests (if necessary);
 #. update or expand the documentation;
+#. update the `CHANGELOG.rst` file with change;
 #. `push <http://rogerdudler.github.io/git-guide/>`_ your feature branch to (your fork of) the matchms repository on GitHub;
 #. create the pull request, e.g. following the instructions `here <https://help.github.com/articles/creating-a-pull-request/>`__.
 
 In case you feel like you've made a valuable contribution, but you don't know how to write or run tests for it, or how to generate the documentation: don't let this discourage you from making the pull request; we can help you! Just go ahead and submit the pull request, but keep in mind that you might be asked to append additional commits to your pull request.
+
```
Comment (jspaaks) [inline CONTRIBUTING.rst]: I expected this at the bottom of the README under a new section _For maintainers_

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -38,7 +39,24 @@ You want to make some kind of change to the code base
...
+You want to make a new release of the code base
+***********************************************
+
+To create release you need write permission on the repository.
+
+1. Check author list in citation.cff and .zenodo.json
```
Comment (jspaaks) [inline CONTRIBUTING.rst]: ``CITATION.cff`` and ``.zenodo.json``

Comment (jspaaks) [reply CONTRIBUTING.rst]: Maybe we should make this a Markdown file?

File: CONTRIBUTING.rst [hunk 3]
```diff
@@ -38,7 +39,24 @@ You want to make some kind of change to the code base
...
+2. Bump the version using `bump2version <major|minor|patch>`
```
Comment (jspaaks) [inline CONTRIBUTING.rst]: double backticks in restructured text

File: CONTRIBUTING.rst [hunk 4]
```diff
@@ -38,7 +39,24 @@ You want to make some kind of change to the code base
...
+3. Update the `CHANGELOG.rst` to include changes made
+4. Goto [GitHub release page](https://github.com/matchms/matchms/releases)
```
Comment (jspaaks) [inline CONTRIBUTING.rst]: link is currently Markdown format

File: CONTRIBUTING.rst [hunk 5]
```diff
@@ -38,7 +39,24 @@ You want to make some kind of change to the code base
...
+5. Press draft a new release button
+6. Fill version, title and description field
+7. Press the Publish Release button
+
+A GitHub action will run which will publish the new version to anaconda.
```
Comment (jspaaks) [inline CONTRIBUTING.rst]: maybe add hyperlink to the place on Anaconda cloud https://anaconda.org/nlesc/matchms

File: CONTRIBUTING.rst [hunk 6]
```diff
@@ -38,7 +39,24 @@ You want to make some kind of change to the code base
...
+Also a Zenodo entry will be made for the release with it's own DOI.
```
Comment (jspaaks) [inline CONTRIBUTING.rst]: ``its`` (because it's used as the possessive here)

**Review by jspaaks — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.rst (hunk 1 above)
Comment (sverhoeven) [reply CONTRIBUTING.rst]: I would rather remove the `Documentation for maintainers` heading in README and keep it here.

File: CONTRIBUTING.rst (hunk 2 above)
Comment (sverhoeven) [reply CONTRIBUTING.rst]: Done

File: CONTRIBUTING.rst (hunk 4 above)
Comment (sverhoeven) [reply CONTRIBUTING.rst]: fixed

**Review by jspaaks — APPROVED** [review]
:+1:

File: .github/workflows/CI_build.yml [hunk 7]
```diff
@@ -257,16 +257,11 @@ jobs:
...
-            --channel conda-forge \
-            --channel nlesc \
-            matchms
           # Install matchms without nlesc channel to prevent package to be installed from nlesc channel instead of wanted $BUILDDIR channel
           conda install \
-            --channel bioconda \
+            --channel $BUILDDIR \
             --channel conda-forge \
-            --channel $BUILDDIR -v \
+            --channel bioconda -v \
```
Comment (hechth) [inline .github/workflows/CI_build.yml:264]: What is the motivation for the change in order? Just asking out of curiosity 👍

**Pull request merged**